recipes/folly/all/conanfile.py

```diff
@@ -105,13 +106,20 @@ def validate(self):
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         extracted_dir = self.name + "-" + self.version
```

Comment:
This is a new feature just to make the recipes easier to work with (no matter how the sources are compressed)

```suggestion
        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```

Reply:
Oh, you also provide `destination` param, now I get it, thanks!